### ISSUE TITLE
Laravel 9

### ISSUE THREAD

**Issue by leasify-it** [rinvex/laravel-subscriptions#184]
Any plans to support L9 in the near future?

**Comment by elasti-co** [comment]
When installing on L9 get error:
`- rinvex/laravel-support v6.1.0 requires felixkiss/uniquewith-validator dev-l9-compatibility -> found felixkiss/uniquewith-validator[dev-master, 1.0.0, ..., 1.1.4, 2.0.0, ..., 2.0.8, 3.0.0, ..., 3.4.2] but it does not match the constraint.
`

**Comment by Omranic** [comment]
Yes, that's because one of the composer dependencies is not yet compatible with Laravel v9, so you've to do one more easy step to get it working, please check this note on upgrading to Laravel v9:

https://github.com/rinvex/laravel-support#note-for-laravel-v9

## Note for Laravel v9

This package requires `felixkiss/uniquewith-validator` dependency, which is not yet compatible with Laravel v9! However as a temporary workaround, add [Laravel Shift](https://github.com/felixkiss/uniquewith-validator/pull/131)'s fork to the `repositories` property of your project root `composer.json`:
```json
{
    "repositories": [
        {
            "type": "vcs",
            "url": "https://github.com/laravel-shift/uniquewith-validator.git"
        }
    ]
}
```
